applications/StructuralMechanicsApplication/custom_utilities/explicit_integration_utilities.h

```diff
@@ -87,6 +92,19 @@ namespace ExplicitIntegrationUtilities
         const double MassFactor
         );
 
+    /**
+     * @brief This method computes the lumping factors of the elements
+     * @param rModelPart The model of the problem to solve
+     * @param LumpingMethod The lumping method considered. The three methods available are:
+     *      - The row sum method
+     *      - Diagonal scaling
+     *      - Evaluation of M using a quadrature involving only the nodal points and thus automatically yielding a diagonal matrix for standard element shape function
+     */
+    void KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ComputeLumpingFactors(
```

Comment:
why in explicit utilities?

Reply:
Because I did know where put it, and the lumping factors are used in explicit utilities many times